qiskit/result/counts.py

```diff
@@ -183,3 +183,11 @@ def int_outcomes(self):
     def _remove_space_underscore(bitstring):
         """Removes all spaces and underscores from bitstring"""
         return int(bitstring.replace(" ", "").replace("_", ""), 2)
+
+    def size(self):
+        """Return the number of clbits"""
+        return len(next(iter(self)))
+
+    def shots(self):
+        """Return the number of shots"""
+        return np.array(list(self.values())).sum()
```

Comment:
Cant you just do `sum(self.values())`? This also assumes there was no loss so every measurement produced an outcome (which is true for our devices but maybe not others)